src/components/calendar/MeetingModal.js

```diff
@@ -0,0 +1,177 @@
+import React, { useState, useEffect } from "react";
+import {
+  MDBContainer,
+  MDBBtn,
+  MDBModal,
+  MDBModalBody,
+  MDBModalHeader,
+  MDBModalFooter,
+  MDBInput,
+  MDBFormInline,
+  MDBIcon
+} from "mdbreact";
+import { DateTimePicker, MuiPickersUtilsProvider } from "@material-ui/pickers";
+
+import { firestore, auth } from "../../config/fbConfig.js";
+import SearchResults from "./SearchResults";
+
+const MeetingModal = props => {
+  const [meeting, setMeeting] = useState({
+    title: "",
+    start: Date.now()
+    // endDate: ""
+  });
+  const [participants, setParticipants] = useState({});
+  const [showSearchResults, setShowSearchResults] = useState(false);
+  const [searchTerm, setSearchTerm] = useState("");
+  const [searchResults, setSearchResults] = useState([]);
+
+  function handleStartDateChange(date) {
+    // console.log('date', date);
+    // console.log('get Hours', date._d.getHours());
+    // console.log('get milliseconds', date._d.getMilliseconds());
+
+    setMeeting({
+      ...meeting,
+      start: date
+    });
+  }
+
+  // function handleEndDateChange(date) {
+  //   changeEndDate(date._d);
+  // }
+
+  const submitMeeting = e => {
+    e.preventDefault();
+    let newParticipants = [];
+    if (participants.length) {
+      newParticipants = participants.map(participant => participant.uid);
+    }
+    newParticipants.push(auth.currentUser.uid);
+    let newMeeting = {
```

Comment:
Checks for selected participants and adds their UID. Then pulls current user's UID and adds it to the metting